src/Common/src/Interop/Windows/winhttp/Interop.SafeInternetHandle.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class WinHttp
+    {
+        // TODO: Move SafeHandleZeroOrMinusOneIsInvalid class to Common
```

Comment:
Is this something you're planning on doing as part of this change, or subsequently?  If we're planning on doing it later, please open an issue to track.